.github/workflows/release.yml

```diff
@@ -5,141 +5,246 @@ on:
       - 'v*'
 
 jobs:
-  build-binary:
-    name: Build binary
+
+  build-package:
+    name: Build package
     runs-on: ubuntu-latest
+    container:
+      image: ubuntu:16.04
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build darwinia
-        env:
-          RUST_TOOLCHAIN: nightly-2021-04-22
+      - name: Install dependencies
         run: |
-          .maintain/setup-darwinia.sh
+          apt update -y
+          apt install --no-install-recommends -y \
+            git curl ca-certificates \
+            make gcc g++ clang-4.0
```

Comment:
This step might be no longer necessary once removing the `container` section.

Reply:
We need to keep using the docker container, so this is required.